configure.ac

```diff
@@ -1053,6 +1053,10 @@ dnl if test "x$linking_mode" != "xdynamic" -a "x$blb_cv_static_openssl" = "xno";
 dnl	AC_MSG_ERROR([static OpenSSL libraries not found (libssl.a, libcrypto.a and their external dependencies like libz.a), either link OpenSSL statically using the --enable-dynamic-linking, or install a static OpenSSL])
 dnl fi
 
+#if defined(TCP_KEEPTIME) && defined(TCP_KEEPIDLE) && defined(TCP_KEEPCNT)
```

Comment:
This is not valid shell/configure.ac code and would result in the macro to be always defined.

Reply:
@bazsi : I tried to search autconf macros, I did not find any `AC_*` field which can check for macro definition,

Reply:
Try the following:

```
AC_COMPILE_IFELSE([AC_LANG_SOURCE([[
#include <netinet/tcp.h>
#ifndef TCP_KEEPIDLE
# error macro not defined
#endif
]])], [AC_DEFINE(`HAVE_TCP_KEEPFILE`, `1`)], [])
```